airbyte-webapp/src/utils/dataPrivacy.ts

```diff
@@ -59,6 +59,12 @@ export const loadOsano = (): void => {
   // Create style element to hide osano widget
   const style = document.createElement("style");
   style.appendChild(document.createTextNode(".osano-cm-widget { display: none; }"));
+  style.appendChild(document.createTextNode(".osano-cm-button--type_denyAll { display: none; }"));
+  style.appendChild(
+    document.createTextNode(
+      `.osano-cm-button--type_manage { background-color: inherit; border: 1px inherit; font-weight: 200; }`
+    )
+  );
```

Comment:
Not really a need to make this individual appendChild calls and create additional text nodes for this, we could just move all those styling into the one string that's already created above, to save some of those DOM nodes from being created. If we turn this into a backtick string, we can just have it multiline, i.e.

```js
style.appendChild(document.createTextNode(`
  .osano-cm-widget { display: none; }
  .osano-cm-button--type_denyAll { display: none; }
  // ...
`));
```